source/bayes_simulation.Rmd

```diff
@@ -45,112 +45,171 @@ correctly identifies 80 percent as "OK" but says that 20 percent are
+that is `(red_cars_broken / 100) / (red_cars / 100)`.  And probabilities would have worked too, given our assumption that the counts are good representations of the yard: `P(red_cars_broken_in_the_yard) / P(red_cars_in_the_yard)`.
 
-`           SCORE a z         `
+**Probability relationships: conditional probability**
 
-Score this number
+We return to the quantity computed above: the **number** of cars being faulty *given* that the mechanic said they were OK.  We use this ratio as an estimate of the probability that that will be true, i.e: the **probability** of cars being faulty *given* that the mechanic said they were OK.
 
-`           END         `
+This is called a *conditional probability*, $P(X | Y)$.  Given a certain outcome Y, calculate the probability of X occurring.
 
-End the IF condition
+Conditional probabilities are calculated as follows:
 
-`           END         `
+$P(X | Y) = P(X \cup Y) / P(Y)$
 
-End REPEAT loop
+From our car example, what is the probability of a car being broken given that a mechanic said it was OK?  It is, out of all the cars that the mechanic said were OK, the fraction that weren't or:
 
-`           COUNT z between 28 30 k         `
+$P(car_broken & mechanic_said_OK) / P(mechanic_said_OK)$.
 
-How many of the SCORED numbers were between "28\--30" (faulty, "OK")
+**Example: conditional probability**
 
-`           SIZE z s         `
+Let's discuss a very relevant example.  You get a COVID test, and the test is negative.  Now, you would like to know what the chance is of you having COVID.
 
-How many numbers were scored
+We have the following information:
 
-`           DIVIDE k s kk         `
+- 1.5% of people in your area have COVID
+- The false positive rate of the tests (i.e., that they detect COVID when it is absent) is very low at 0.5%
+- The false negative rate (i.e., that they fail to detect COVID when it is present) is quite high at 40%
 
-What proportion were faulty, "OK"
+You would like to know the probability of having COVID given that your test was negative.  Using the conditional probability relationship from above, we can write:
 
-`           PRINT kk         `
+$P(covid | test_negative) = P(covid \cup test_negative) / P(test_negative)$
 
-Print result
+We notice that $P(covid \cup test_negative) = P(test_negative | covid) * P(covid) = .4 * .015 = 0.006$.
+We also observe that $P(test_negative) = P(test_negative | covid) + P(test_negative | not_covid) = 0.4 + 0.995 = 1.395$.
 
-Result kk = 0.039
+We fill in the quantities above and obtain:
 
-Hence, we estimate that the probability that a car identified as "OK"
-will really be faulty is .039.
+$P(covid | test_negative) = 0.006 / 1.395 = 0.004$.
+
+OK, so we have a number... is that number even close to correct?  Doing the simulation is a lot easier than figuring out the calculation above:
```

Comment:
It should be 0.006 / (0.006 + P(T- ^ C-)

P(T- ^ C-) = P(T- | C-) P(C-) = 0.995 * 0.85

0.006 / (0.006 +  0.995 * 0.85) = 0.007

Reply:
We *want* to calculate the conditional probability: have covid GIVEN THAT test negative.  But I think it's fixed in the text now.